### ISSUE TITLE
Jquery or a plain Javascript version

### ISSUE THREAD

**Issue by weedkiller** [frontend-collective/react-sortable-tree#321]
Hello, any chance you would consider making this available to plain JS or Jquery

**Comment by wuweiweiwu** [comment]
@weedkiller yep! I'm planning to add a global friendly `umd` build, probably in a minor release

**Comment by claudius108** [comment]
Hi,

Any news with this umd build?

Claudius
